src/test/java/Homework23.java

```diff
@@ -0,0 +1,82 @@
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import pages.HomePage;
+import pages.LoginPage;
+import pages.PlaylistPage;
+
+public class Homework23 extends BaseTest{
+    HomePage homePage;
+    PlaylistPage playlistPage;
+    LoginPage loginPage;
+    String newName = "Updated playlist \"Rename.\"";
+    String namedPlaylist = "Created playlist \"Test Pro.\"";
+    String notification = "Deleted playlist \"Test Pro.\"";
+    @Test
+       public void login() {
+        loginPage = new LoginPage(driver);
+        homePage = new HomePage(driver);
+
+        loginPage.loginCorrectMethod();
+        Assert.assertTrue(homePage.avatarDisplayed());
+    }
+
+    @Test
+    public void renamePlaylist() {
+        loginPage = new LoginPage(driver);
+        playlistPage = new PlaylistPage(driver);
+
+        loginPage.loginCorrectMethod();
+        playlistPage.clickPlaylist()
+            .rightCLickPlaylist()
+            .clickEdit()
+            .enterNewName("Rename1");
```

Comment:
Here you enter new name as Rename1 but at step verify notification return you verify with name is Rename `String newName = "Updated playlist \"Rename.\"";` 
so test failed at step assertion